content/en/synthetics/guide/browser-tests-passkeys.md

```diff
@@ -0,0 +1,61 @@
+kind: guide
+description: Learn how to ensure your Synthetic browser tests can log in to your applications. 
+further_reading:
+  - link: '/synthetics/guide/app-that-requires-login/'
+    tag: 'Documentation'
+    text: 'Learn more about authentication in browser test'
+  - link: '/synthetics/guide/browser-tests-totp'
+    tag: 'Documentation'
+    text: 'TOTPs for Multi-Factor Authentication (MFA) in browser tests'
+  - link: 'synthetics/settings/?tab=specifyvalue#global-variables'
+    tag: 'Documentation'
+    text: 'Learn more about global variables'
+---
+
+## Overview
+
+Passkeys (FIDO2) offer stronger security than the standard username and password tuple, and rely on cryptographic login credentials that are unique across every website. passkeys never leave your user's devices and are never stored on a web application server. This security model eliminates the risks of phishing, all forms of password theft, and replay attacks.
+
+You can use passkeys as a replacement for a username and password or as a second factor authentication. Synthetic Monitoring can now generate, store and leverage passkeys to test your application’s passkeys-based authentication modules and critical user journeys without disabling this critical security measure.
+
+## Create your Virtual Authenticator global variable
+
+Passkeys in Synthetic Monitoring are handled by Virtual Authenticator Global Variables. 
+To create a Virtual Authenticator global variable storing your passkeys, see the [**Global Variables** section in Synthetic Monitoring & Continuous Testing Settings][4].
+
+{{< img src="synthetics/guide/browser-tests-passkeys/new-variable-fido2.png" alt="Create a FIDO2 key" style="width:70%;" >}}
+
+## Use passkeys in your Synthetic tests
+When [creating a browser test][3], complete your application's passkeys login/registering flow using the passkeys stored in your virtual authenticator global variable.
+**Note:** Today passkeys are only supported in Chrome and Edge Synthetics browser tests.
+
+### Test a registration flow
+
+To test a registration flow using passkeys in your [browser tests][3]:
+
+1. Import your Virtual Authenticator global variable into your test.[Documentation][5] 
+2. Navigate to the page to register your passkey. When recording your test, Datadog automatically completes any passkey registration challenge by generating a passkey using the virtual authenticator. The passkey will then be store on Datadog.
+3. After recording your test steps, click **Save & Launch Test**.
+
+### Test a login flow
+
+To test a login flow using a passkey in your [browser tests][3], you need to have your Datadog Passkey registered on the web application (see section above). This is required once per key and application.
+
+**Note:**
+This can be done either by completing the registration form from within the recorder without recording the registration steps, or by creating a test that embeds both the recording and the login.
```

Comment:
```suggestion
This can be done in two ways:
- Either by completing once the registration flow from within the recorder, but without recording the registration steps,
- Or by creating a test that embeds both steps for the registration and login flows.
```